README.md

```diff
@@ -40,10 +40,6 @@ MINGW64 terminal:
 pacman -Syu --needed base-devel git mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl
 ```
 
-Once done, add the MINGW64 bin folder to the system PATH variable. If you've
-installed MSYS2 using the default installation settings, the bin folder path
-should be `C:\msys64\mingw64\bin`.
-
```

Comment:
Do you still need this for the static linking?

Reply:
Removed.

Reply:
But the build won't work if the user doesn't add the path, so this needs to be returned?

Reply:
No, it should work because the mingw console adds mingw64/bin to the `PATH`.